wiki/Beatmap/Beatmap_tags/en.md

```diff
@@ -0,0 +1,216 @@
+
+These keywords can then be set as a filter in [song select](/wiki/Client/Interface#song-select) as well as the [beatmap listing](https://osu.ppy.sh/beatmapsets) via full-text search.
+
+## User tags
+
+*Note: This feature is not available in osu!(stable).*
+
+**User tags** are tags that describe specific characteristics of a beatmap, such as its patterns or required skill sets. After achieving at least a C [grade](/wiki/Gameplay/Grade) on a beatmap, users are able to select and vote for user tags in the result screen in osu!(lazer). Upon receiving at least 10 upvotes, the respective user tag will be displayed in the beatmap information page.
+
+To filter for beatmaps with certain user tags, click on the corresponsing tag in the beatmap information page, or type in `tag="{name}"` in the beatmap listing where `{name}` is the name of any tag listed below.
+
+The tables below list all user tags grouped by category as well as the game modes they are applicable in.
+
+### Aim
+
+| Tag name | Description | Game mode |
+| :-- | :-- | :-- |
+| aim/aim control | Patterns with velocity or direction changes which strongly go against a player's natural movement pattern. | ![][osu!] |
+| aim/flow aim | Patterns with fully continuous cursor movement, usually due to a combination of wide angles and little time between objects. | ![][osu!] |
+| aim/jump aim | Focuses heavily on jumps, i.e. circles spaced far apart that require the player to move towards, slow down to hit, then speed up to move towards the next object. | ![][osu!] |
+| aim/linear aim | Patterns, usually jumps, that require the player to move continuously in a straight or nearly straight line. | ![][osu!] |
+| aim/precision | Colloquial term for maps which require fine, precise movement to aim correctly. Typically refers to maps with circle sizes above and including 6. | ![][osu!] |
+| aim/precision | Colloquial term for maps which require fine, precise movement to catch fruits. Typically refers to maps with circle sizes above and including 6. | ![][osu!catch] |
+| aim/sharp aim | Patterns with heavy use of sharp angle movement, either between circles or into sliders. | ![][osu!] |
+| aim/spaced jumps | Focuses heavily on jumps with a lot of spacing. | ![][osu!] |
+| aim/squares | Focuses heavily on jumps that follow square patterns. | ![][osu!] |
+| aim/stamina | Tests a player's ability to aim spaced jumps over long periods of time. | ![][osu!] |
+| aim/triangles | Focuses heavily on jumps that follow triangular patterns. | ![][osu!] |
+| aim/wide aim | Patterns, usually jumps, with heavy use of wide angle movement. | ![][osu!] |
+
+### Geometric
+
+| Tag name | Description | Game mode |
+| :-- | :-- | :-- |
+| geometric/grid snap | Objects are placed along a square grid, typically using osu!'s in-built grid snap feature. | ![][osu!] |
```

Comment:
"built-in"?

Reply:
When writing the article, I asked this myself as well. I found [a thread](https://english.stackexchange.com/questions/129360/is-inbuilt-a-word-is-it-alright-to-use-it-or-should-i-use-built-in) stating that `in-built` is used in the UK.

 [Another thread](https://english.stackexchange.com/questions/1250/built-in-or-in-built) says that `built-in` is the much more common version of the two words.

Also related to https://github.com/ppy/osu-wiki/pull/13171#issuecomment-2825326809

Reply:
```suggestion
| geometric/grid snap | Objects are placed along a square grid, typically using osu!'s built-in grid snap feature. | ![][osu!] |
```